### ISSUE TITLE
Error on media stream videotracks on IOS build still persists 

### ISSUE THREAD

**Issue by scorp-dmitriy** [OpenVidu/openvidu#529]
**Describe the bug**
I'm using the iPad with iPadOS 13.6.1
The issue described [here](https://github.com/OpenVidu/openvidu/issues/490) is still not fixed. I wasn't able to build the app until I've overridden the method with [this](https://github.com/OpenVidu/openvidu/issues/490#issuecomment-633705637).

**Expected behavior**
The video stream should work on iPad Pro.
Looks like this it's needed to add this condition
```typescript
// Accept: Ipad PRO
if (/Macintosh/.test(navigator.userAgent) && 'ontouchend' in document) {
   return 1;
}
```

**Wrong current behavior**
The app exists with ` ERROR: There was an error connecting to the session: Browser null (version null) for OS X is not supported in OpenVidu.`

**OpenVidu tutorial where to replicate the error**
1. Clone openvidu-tutorials repository
2. Compile the application using ionic cordova emulate ios --device
4. Run the application on the iPad
6. See error when joining the session

**Comment by rgustavsson** [comment]
However, openvidu does identify my iPadPro as an iPad via the `isIPhoneOrIPad` method, we just get screwed on the second clause which is `isIOSWithSafari || platform['isIonicIos']`

I believe that the same iPad check should be done when we set `platform['isIonicIos'] `

So a fix could be:

```
const isTouchable = 'ontouchend' in document;
const isIPad = /\b(\w*Macintosh\w*)\b/.test(platform.ua) && isTouchable;
platform['isIonicIos'] = (platform.product === 'iPhone' || platform.product === 'iPad' || (isIPad && isTouchable )) && platform.ua!!.indexOf('Safari') === -1;
```

**Comment by rgustavsson** [comment]
#531

**Comment by CSantosM** [comment]
Fixed on master (https://github.com/OpenVidu/openvidu/pull/550). 
It'll be included in the next release (2.16.0)
